gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystem.java

```diff
@@ -1093,53 +1096,50 @@ private GoogleCloudStorageItemInfo getFileInfoInternal(
     }
     StorageResourceId dirId = resourceId.toDirectoryId();
 
-    ExecutorService dirExecutor =
-        options.isStatusParallelEnabled()
-            ? resourceId.isDirectory()
-                ? Executors.newSingleThreadExecutor(DAEMON_THREAD_FACTORY)
-                : Executors.newFixedThreadPool(2, DAEMON_THREAD_FACTORY)
-            : new LazyExecutorService();
```

Comment:
I think that we only need to replace executors here, and revert rest of the changes.